deid/dicom/actions.py

```diff
@@ -23,47 +23,36 @@
 """
 
 from deid.logger import bot
-from deid.dicom.tags import update_tag
 from deid.utils import get_timestamp
 
-import re
-
-
 # Timestamps
 
 
-def jitter_timestamp(dicom, field, value):
+def jitter_timestamp(field, value):
     """if present, jitter a timestamp in dicom
     field "field" by number of days specified by "value"
     The value can be positive or negative.
 
     Parameters
     ==========
-    dicom: the pydicom Dataset
     field: the field with the timestamp
     value: number of days to jitter by. Jitter bug!
 
     """
     if not isinstance(value, int):
         value = int(value)
 
-    original = dicom.get(field)
+    original = field.element.value
```

Comment:
Could this be none first? (field.element)

Reply:
I don't think so.  I checked the places that create DicomField objects, and they're checking to ensure that element is a DataElement prior to the creation of the DicomField.